lms/resources/_js_config/__init__.py

```diff
@@ -198,6 +198,25 @@ def enable_lti_launch_mode(self, assignment: Assignment):
         }
         self._config["debug"]["values"] = self._get_lti_launch_debug_values()
 
+        self._config["editing"] = {
```

Comment:
With every launch we send the information of the endpoint to get the assignment current config + file pickers info back.